Flow-Control/Busy-Wait-Button/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include "mbed.h"
+
+DigitalIn button(SW1); // Change to match your board
+DigitalOut led(LED1);
+
+#define button_press 0
```

Comment:
Isn't convention to have all #defines in CAPS?

Reply:
@cmonr Probably, I just copied/pasted from the old main.cpp. I'll fix it